src/data/services/LmsApiService.js

```diff
@@ -384,6 +385,57 @@ class LmsApiService {
     const url = `${LmsApiService.baseUrl}/enterprise/api/v1/analytics-summary/${enterpriseUUID}`;
     return LmsApiService.apiClient().post(url, formData);
   }
+
+  static updateUserActiveEnterprise = (enterpriseId) => {
```

Comment:
nit: might co-locate this function with the others related to active enterprise (i.e., `fetchEnterpriseLearnerData`, `getActiveLinkedEnterprise`).

Reply:
with the removal of loginRefresh they're already co-located! 👍